tests/test_iso_parsing.py

```diff
@@ -595,3 +595,32 @@ def test_md_parsing_keywords_no_anchor():
     assert iden.keywords[1].keywords[1].url is None
     assert iden.keywords[1].keywords[2].name == 'parcelles cadastrales'
     assert iden.keywords[1].keywords[2].url is None
+
+def test_md_indentifier_anchor():
+    """Test the parsing of MD_Keywords where the keyword is not defined by a 
+    gmx:Anchor
+    
+    MD_Metadata record available in
+    tests/resources/csw_geobretagne_mdmetadata.xml
+
+    """
+    md_resource = get_md_resource(
+        'tests/resources/csw_iso_identifier.xml')
+    md = MD_Metadata(md_resource)
+    assert type(md) is MD_Metadata
+    
+    
+    # gmd:referenceSystemInfo/gmd:MD_ReferenceSystem/gmd:referenceSystemIdentifier/gmd:RS_Identifier/gmd:code/gmx:Anchor = 'http://www.opengis.net/def/crs/EPSG/0/4937'
+    assert md.referencesystem.code == 'ETRS89-GRS80'
+    # gmd:identifier/gmd:MD_Identifier/gmd:code/gmx:Anchor = 'https://www.nationaalgeoregister.nl/geonetwork/srv/metadata/f44dac86-2228-412f-8355-e56446ca9933'
+    iden = md.identification[0]
+    assert_list(iden.uricode, 1)
+    assert iden.uricode[0] == 'https://www.nationaalgeoregister.nl/geonetwork/srv/metadata/f44dac86-2228-412f-8355-e56446ca9933'
+    
+    # gmd:pointOfContact/gmd:CI_ResponsibleParty/gmd:organisationName/gmx:Anchor = 'http://standaarden.overheid.nl/owms/terms/Ministerie_van_Defensie'
```

Comment:
Can these lines be removed?

Reply:
Hi Seth, it is a reminder of upcoming work, but if you prefer I can remove them